man/systemd.network.xml

```diff
@@ -3208,7 +3216,30 @@
         <term><varname>Priority=</varname></term>
         <listitem>
           <para>Specifies the priority of the class. In the round-robin process, classes with the lowest
-          priority field are tried for packets first. This setting is mandatory.</para>
+          priority field are tried for packets first.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Quantum=</varname></term>
+        <listitem>
+          <para>Specifies how much bytes to serve from leaf at once.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>MTUBytes=</varname></term>
+        <listitem>
+          <para>Specifies the maximum packet size we create. When suffixed with K, M, or G, the specified
+          size is parsed as Kilobits, Megabits, or Gigabits, respectively, to the base of 1024.</para>
```

Comment:
The option has "Bytes" in its name but size suffixes are interpreted as *bits*? very confusing

Reply:
Ouch.